.github/workflows/scripts/packageChart.js

```diff
@@ -45,6 +45,17 @@ module.exports = async ({ core, exec }) => {
         }
       }
 
+      // Run Helm unit tests if enabled
+      if (chart.tests) {
+        try {
+          core.notice(`Packaging the ${chart.name} chart`)
+          await exec.exec('helm', ['plugin', 'install', '"https://github.com/helm-unittest/helm-unittest"', '--version', '0.4.1'], { cwd: `${checkoutSourceDir}/${chart.source}` })
```

Comment:
hardcoding the version here does not sound great — I'm worried we will forget about it and never update it